security-openid-connect-quickstart/pom.xml

```diff
@@ -32,6 +32,10 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
```

Comment:
@survivant Why is this module added here ? There is a dedicated `keycloak-authorization-quickstart`

Reply:
@sberyozkin maybe it's not necessary.  I though it was used by KeycloakTestClient.  So the sample : **security-keycloak-authorization-quickstart** need that module to check in Keycloak if a URL is authorized for the user logged.  

and the sample : **security-openid-connect-quickstart**   will received a token with the roles in it and use that to validate if the user is authorized with the annotation @RolesAllowed so the dependency to keycloak autorisation is not needed ?

did I understand correctly the difference between the two samples ?